src/jvm/main/org/jetbrains/kotlinx/lincheck/CTestConfiguration.kt

```diff
@@ -64,28 +62,4 @@ abstract class CTestConfiguration(
         const val DEFAULT_MINIMIZE_ERROR = true
         const val DEFAULT_TIMEOUT_MS: Long = 10000
     }
-}
-
-internal fun createFromTestClassAnnotations(testClass: Class<*>): List<CTestConfiguration> {
```

Comment:
You still need this code; both the annotations and the old `Option`s must work until the next release. We will remove them after that, cleaning up the codebase accordingly.